rotkehlchen/rotkehlchen.py

```diff
@@ -336,13 +336,30 @@ def deactivate_premium_status(self) -> None:
         self.premium_sync_manager.premium = None
         self.chain_manager.deactivate_premium_status()
 
+    def premium_sync_data(
```

Comment:
No need for this function if we remove the name argument. From the rest api call directly into the premium object.